tox.ini

```diff
@@ -60,46 +60,24 @@ deps =
 commands =
   charm: pyright
 
-[testenv:unit]
-description = Run unit tests
+[testenv:{unit, scenario}]
```

Comment:
What is the effect of this new syntax?
Why not `[testenv:unit]` if it just runs both of them anyway? (does it?)

Reply:
it does. and the effect of the syntax is to have two synonim envs instead of copy-pasting them. on the long run I'd like to drop the "scenario" one altogether and have all tests in /unit, but our CI ATM will still attempt to run `tox -e scenario` and `tox -e unit` so we still need to expose both envs for now